Assets/Scripts/StandardSamples/UI/Storage/Title/UITitleStorageMenu.cs

```diff
@@ -227,5 +232,19 @@ public void UpdateFileContent(Result result)
                 Debug.LogErrorFormat("UITitleStorageMenu - '{0}' file content was not found in cached data storage.", FileNameTextBox.InputField.text);
             }
         }
+
+        public void MustHaveAtLeastOneTagValidator(SelectableStateHandler toUpdate)
```

Comment:
![image](https://github.com/user-attachments/assets/6ccedccc-771b-45d2-af5f-aa22461dd6ab)

Is Visual Studio lying to me - or is this function no longer utilized?

Reply:
It's only referenced _in the Editor_, not in the csproj. It's referenced in a UnityEvent.

Exactly what you're experiencing right now was the most prominent argument for not doing it the way I did. One counterpoint is that if you used Rider instead of Visual Studio, it'd tell you where it was use. The other is that all things like this will have the same argument, showing what kind of thing it's related to.

Reply:
I recall that now. Okay.